subsys/net/lib/zzhc/zzhc.c

```diff
@@ -581,26 +590,20 @@ static void zzhc_init(void *arg1, void *arg2, void *arg3)
 	struct zzhc *ctx;
 
 	/* Unconditionally subscribe +CEREG events */
-	if (zzhc_at_cmd_xfer("AT+CEREG=5", NULL, 0) != 0) {
+	if (nrf_modem_at_printf("AT+CEREG=5") != 0) {
 		LOG_DBG("Can't subscribe to +CEREG events.");
 		return;
 	}
 
 	/* Unconditionally subscribe %XSIM events */
-	if (zzhc_at_cmd_xfer("AT%XSIM=1", NULL, 0) != 0) {
+	if (nrf_modem_at_printf("AT%%XSIM=1") != 0) {
 		LOG_DBG("Can't subscribe to %%XSIM events.");
 		return;
 	}
 
-	/* Allocate resource. */
-	ctx = (struct zzhc *)zzhc_malloc(sizeof(struct zzhc));
-	if (ctx == NULL) {
-		LOG_DBG("Can't allocate memory for context.");
-		return;
-	}
+	ctx = &zzhc_context;
```

Comment:
If we use a BSS data here, we must remove the line 637, too
	zzhc_free(ctx);
Otherwise k_free(address in BSS) will trigger crash.